### PR TITLE
API documentation

### PR DESCRIPTION
**Pull request by rtxm**
See Netlify preview.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,14 +3,20 @@
 The Uh!live Python SDK provides convenient access to the Uh!live API from
 applications written in the Python language.
 
-Read the [documentation for the Conversation API](https://docs.allo-media.net/live-api/) and [for the Recognition API (vocal bot toolkit)](https://docs.allo-media.net/stream-api-bots/).
+Read the [full documentation](https://python-uhlive-sdk.netlify.app/).
```
Comment (shostelet) [inline README.md:6]: Why? I can't access this website

File: docs/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+from uhlive.auth import build_authentication_request
+import requests
+
+uhlive_client = "…"
+uhlive_secret = "…"
+# user_id = "…"
+# user_password = "…"
+
+auth_url, auth_params = build_authentication_request(uhlive_client, uhlive_secret)
+# or auth_url, auth_params = build_authentication_request(uhlive_client, user_id=user_id, user_pwd=user_password)
```
Comment (shostelet) [inline docs/index.md:31]: this is the old auth way? shouldn't we get rid of it as it is deprecated?

File: README.md (hunk 1 above)
Comment (rtxm) [reply README.md:6]: you're just reviewing it as a netlify preview before deployment :wink: but I already put the link in anticipation.

File: docs/index.md (hunk 2 above)
Comment (shostelet) [reply docs/index.md:31]: both line 30 and 31 are KC connection? If so, ignore my comment

Comment (rtxm) [reply docs/index.md:31]: both are KC connections indeed.

File: README.md (hunk 1 above)
Comment (merwan) [reply README.md:6]: We should create a DNS entry for this documentation. Something like https://docs.uh.live/sdk/python.
@shostelet I let you think about a proper structure for the URL

Comment (shostelet) [reply README.md:6]: Of course, I was not yet fully awake :sweat_smile: 
https://docs.uh.live/sdk/python suits me

Comment (rtxm) [reply README.md:6]: It must be a subdomain to work with Netlify, not a path @merwan

Comment (merwan) [reply README.md:6]: We can use Netlify rewrite rules to setup such URL: https://docs.netlify.com/routing/redirects/rewrites-proxies/#proxy-to-another-netlify-site
I will try to setup https://docs.allo-media.net/sdk/python like this

**Review by merwan — APPROVED** [review]
I did not read the whole documentation, but this seems very complete, thanks!

**Pull request merged**